### DISCUSSION TITLE
今まさにチームメンバーが何のタスクに着手しているかを，確認できる仕組みが欲しい

### DISCUSSION THREAD

**Discussion by ntk221** [ntk221/minishell#48] [Ideas]
この仕組みがあれば，サボり防止になるかも😄

**Comment by nekotonatsu** [comment]
たしかに！

**Comment by ntk221** [comment]
Slack 使うのもありかもしれないです🕵️‍♂️

https://qiita.com/unsoluble_sugar/items/e42ac352400598a24915

https://zenn.dev/kino_puehddby/articles/d6172cb5503166
